<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmkOptionDialog</class>
 <widget class="QDialog" name="SmkOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>509</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Smark Option</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc/icon.qrc">
    <normaloff>:/icon/file-option.png</normaloff>:/icon/file-option.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>208</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Consolas</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabGui">
      <attribute name="title">
       <string>Gui</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelStyle">
         <property name="text">
          <string>Qt Gui Style:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="comboBoxGuiStyle"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelFont">
         <property name="text">
          <string>Editor Font:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFontComboBox" name="comboBoxEnglishFont"/>
       </item>
       <item row="1" column="2">
        <widget class="QFontComboBox" name="comboBoxChineseFont"/>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spinBoxFontSize"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelColor">
         <property name="text">
          <string>Syntax Color:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QListWidget" name="listColor"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabContext">
      <attribute name="title">
       <string>Context</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="labelCSS">
         <property name="text">
          <string>CSS URL:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditCSSURL"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="buttonCSSURL">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelMathJax">
         <property name="text">
          <string>MathJax URL:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditMathJaxURL"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="buttonMathJaxURL">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabHeader">
          <attribute name="title">
           <string>HTML Header</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="labelHeader">
             <property name="text">
              <string>Insert Context between &lt;header&gt; and &lt;/header&gt; :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPlainTextEdit" name="textEditHeader">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabHat">
          <attribute name="title">
           <string>HTML Hat</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="labelHat">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Insert Context after &amp;lt;body&amp;gt; :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPlainTextEdit" name="textEditHat">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabTail">
          <attribute name="title">
           <string>HTML Tail</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="labelTail">
             <property name="text">
              <string>Insert Context before &lt;/body&gt;:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPlainTextEdit" name="textEditTail">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabFoot">
          <attribute name="title">
           <string>HTML Foot</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="labelFoot">
             <property name="text">
              <string>Insert Context after &lt;/body&gt; :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPlainTextEdit" name="textEditFoot">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rc/icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SmkOptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SmkOptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
